4-js/index.js

```diff
@@ -0,0 +1,330 @@
+const userAddress = document.getElementById('user-address');
+const userCity = document.getElementById('user-city');
+const userCountry = document.getElementById('user-country');
+const formUser = document.getElementsByClassName('form-user')[0];
+const editUserBtn = document.getElementsByClassName('edit-user')[0];
+const createUserBtn = document.getElementsByClassName('create-user')[0];
+const addUserBtn = document.getElementsByClassName('add-user')[0];
+const allDelete = document.getElementById('all-delete');
+const arrUserCheckbox = document.getElementsByClassName('user-checkbox');
+const companyInput = document.getElementById('company-input');
+const nameInput = document.getElementById('name-input');
+const addressInput = document.getElementById('address-input');
+const inputFilter = document.getElementsByClassName('input-filter');
+const titleNewUser = document.getElementById('title-new-user');
+const titleEditUser = document.getElementById('title-edit-user');
+const paginator = document.getElementById('paginator');
+const paginations = document.getElementsByClassName('paginator-number');
+
+document.addEventListener('DOMContentLoaded', () => {
+  findArrLenghtUsers();
+  addInputListener();
+});
+
+addUserBtn.addEventListener('click', () => {
+  const arrFormUserInput = document.getElementsByClassName('form-user-input');
+  formUser.classList.remove('is-hide');
+  createUserBtn.classList.remove('is-hide');
+  titleNewUser.classList.remove('is-hide');
+  createUserBtn.setAttribute('disabled', 'disabled');
+  createUserBtn.classList.add('is-translucent');
+  for (let i = 0; i < arrFormUserInput.length; i++) {
+    arrFormUserInput[i].addEventListener('keyup', formCheck);
+  }
+});
+
+createUserBtn.addEventListener('click', () => {
+  createUser();
+  formUser.classList.add('is-hide');
+  createUserBtn.classList.add('is-hide');
+  titleNewUser.classList.add('is-hide');
+  userCompany.value = '';
+  userName.value = '';
+  userAddress.value = '';
+  userCity.value = '';
+  userCountry.value = '';
+});
+
+allDelete.addEventListener('click', () => {
+  deleteSelectedUsers();
+});
+
+mainCheckbox.addEventListener('click', () => {
+  for (let i = 0; i < arrUserCheckbox.length; i++) {
+    if (mainCheckbox.checked !== arrUserCheckbox[i].checked) {
+      arrUserCheckbox[i].checked = !arrUserCheckbox[i].checked;
+    }
+  }
+  allDelete.classList.toggle('is-hide');
+});
+
+editUserBtn.addEventListener('click', () => {
+  editUser();
+  userCompany.value = '';
+  userName.value = '';
+  userAddress.value = '';
+  userCity.value = '';
+  userCountry.value = '';
+  formUser.classList.add('is-hide');
+  editUserBtn.classList.add('is-hide');
+  titleEditUser.classList.add('is-hide');
+});
+
+function findArrLenghtUsers() {
+  sendRequsest('GET',requestUrl).then((data) => {
+    createPaginator(data.length)
+  })
+  .catch((err) => console.log(err));
+}
+
+function createPaginator(length) {
+  const quantityPagination = Math.ceil(length / 10);
```

Comment:
10 must be in defined in constant, because it is a limit of items per page and you use it several times here

Reply:
done